protocol/communities/community.go

```diff
@@ -377,27 +375,18 @@ func (o *Community) MarshalJSONWithMediaServer(mediaServer *server.MediaServer)
 		communityItem.Permissions = o.config.CommunityDescription.Permissions
 		communityItem.IntroMessage = o.config.CommunityDescription.IntroMessage
 		communityItem.OutroMessage = o.config.CommunityDescription.OutroMessage
-
-		// update token meta image to url rather than base64 image
-		var tokenMetadata []*protobuf.CommunityTokenMetadata
-		for _, m := range o.config.CommunityDescription.CommunityTokensMetadata {
-			copyM := proto.Clone(m).(*protobuf.CommunityTokenMetadata)
-			copyM.Image = mediaServer.MakeCommunityDescriptionTokenImageURL(o.IDString(), copyM.GetSymbol())
-			tokenMetadata = append(tokenMetadata, copyM)
-		}
-		communityItem.CommunityTokensMetadata = tokenMetadata
-
+		communityItem.CommunityTokensMetadata = o.config.CommunityDescription.CommunityTokensMetadata
 		communityItem.ActiveMembersCount = o.config.CommunityDescription.ActiveMembersCount
 
 		if o.config.CommunityDescription.Identity != nil {
 			communityItem.Name = o.Name()
 			communityItem.Color = o.config.CommunityDescription.Identity.Color
 			communityItem.Description = o.config.CommunityDescription.Identity.Description
-			for t := range o.config.CommunityDescription.Identity.Images {
+			for t, i := range o.config.CommunityDescription.Identity.Images {
```

Comment:
Alternatively, these can be moved to MarshalJSON (communityItem.Images and communityItem.CommunityTokensMetadata)